companion.py

```diff
@@ -488,22 +563,28 @@ def fixup(data):
 
 # Return a subset of the received data describing the current ship
 def ship(data):
-
     def filter_ship(d):
         filtered = {}
         for k, v in d.items():
             if v == []:
-                pass	# just skip empty fields for brevity
-            elif k in ['alive', 'cargo', 'cockpitBreached', 'health', 'oxygenRemaining', 'rebuilds', 'starsystem', 'station']:
-                pass	# noisy
-            elif k in ['locDescription', 'locName'] or k.endswith('LocDescription') or k.endswith('LocName'):
-                pass	# also noisy, and redundant
-            elif k in ['dir', 'LessIsGood']:
-                pass	# dir is not ASCII - remove to simplify handling
+                pass  # just skip empty fields for brevity
+
+            elif k in ('alive', 'cargo', 'cockpitBreached', 'health', 'oxygenRemaining',
+                       'rebuilds', 'starsystem', 'station'):
+                pass  # noisy
+
+            elif k in ('locDescription', 'locName') or k.endswith('LocDescription') or k.endswith('LocName'):
+                pass  # also noisy, and redundant
+
+            elif k in ('dir', 'LessIsGood'):
+                pass  # dir is not ASCII - remove to simplify handling
+
```

Comment:
This can probably be replaced with a `filter()` and a few constants.

Reply:
Looking at this some more, while it could be done that way, it wouldnt be any cleaner.